Package.swift

```diff
@@ -131,9 +135,11 @@ extension Target.Dependency {
   )
   static let dequeModule: Self = .product(name: "DequeModule", package: "swift-collections")
   static let atomics: Self = .product(name: "Atomics", package: "swift-atomics")
+  static let tracing: Self = .product(name: "Tracing", package: "swift-distributed-tracing")
 
   static let grpcCore: Self = .target(name: "GRPCCore")
   static let grpcInProcessTransport: Self = .target(name: "GRPCInProcessTransport")
+  static let grpcInterceptors: Self = .target(name: "GRPCInterceptors")
```

Comment:
Hmm well, we'll likely have tracing, logging and metrics interceptors. What other kinds of extras would end up in an extras package? I agree it may be too small a unit, but my only concern with an `extras` module is that it will end up being a bag for random things if it's not well defined. Of course this seems to have worked well (?) in our other repos, so maybe this is a non-issue :)